src/github.js

```diff
@@ -5,6 +5,18 @@ export default class GitHub {
 	constructor() {
 		const token = core.getInput("token") || process.env.GITHUB_TOKEN || "";
 		this.octokit = github.getOctokit(token);
+
+		const formats = core.getInput("format").replace(/ /g, '').split(',').map(function(value){
+			return value.trim();
+		}) || "git";
```

Comment:
The way I read this is that formats will either be an array that is the outcome of the split -> map, or it will be the string "git". I might be confused or this line might be confusing.

Reply:
Yes, the default should be an array which includes `git` as a value. I've updated.